components/esptool_py/Kconfig.projbuild

```diff
@@ -103,6 +103,8 @@ choice ESPTOOLPY_FLASHSIZE
 	help
 		SPI flash size, in megabytes
 
+config ESPTOOLPY_FLASHSIZE_1MB
+	bool "1 MB"
```

Comment:
Modification for 1MB is not related with this PR.

Maybe we will not recommend to use 1MB flash, since the binary size is increased, and maybe larger than 512KB, 1MB flash cannot support OTA.